prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -38,7 +38,7 @@ class FakeReport {
                 }
                 Element.Type.DATETIME -> {
                     val date = faker.date().past(10, TimeUnit.DAYS)
-                    val formatter = SimpleDateFormat("yyyyMMddhhmm")
+                    val formatter = SimpleDateFormat("yyyyMMdd")
```

Comment:
This change is per AZ request

Reply:
The default should still be "yyyyMMddhhmm" because that follows national standards (HL7 ...). I suggest that we use the format field of the Element and stuff "yyyyMMdd" in there. So the code should be:
```
SimpleDateFormat(element.format ?: "yyyyMMddhhmm")
```

Reply:
will do